## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 514264`
Commit: `305b5672682d75294e2dd05da94a1238810637aa`

---
### `x86_64-linux`
<details>
  <summary>:x: 68 packages failed to build:</summary>
  <ul>
    <li>mlflow-server</li>
    <li>mlflow-server.dist</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>python313Packages.databricks-sdk</li>
    <li>python313Packages.databricks-sdk.dist</li>
    <li>python313Packages.executorch</li>
    <li>python313Packages.executorch.dist</li>
    <li>python313Packages.langchain-azure-dynamic-sessions</li>
    <li>python313Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python313Packages.langchain-classic</li>
    <li>python313Packages.langchain-classic.dist</li>
    <li>python313Packages.langchain-community</li>
    <li>python313Packages.langchain-community.dist</li>
    <li>python313Packages.langchain-deepseek</li>
    <li>python313Packages.langchain-deepseek.dist</li>
    <li>python313Packages.langchain-experimental</li>
    <li>python313Packages.langchain-experimental.dist</li>
    <li>python313Packages.langchain-mongodb</li>
    <li>python313Packages.langchain-mongodb.dist</li>
    <li>python313Packages.langchain-openai</li>
    <li>python313Packages.langchain-openai.dist</li>
    <li>python313Packages.langchain-xai</li>
    <li>python313Packages.langchain-xai.dist</li>
    <li>python313Packages.langgraph-checkpoint-mongodb</li>
    <li>python313Packages.langgraph-checkpoint-mongodb.dist</li>
    <li>python313Packages.langgraph-store-mongodb</li>
    <li>python313Packages.langgraph-store-mongodb.dist</li>
    <li>python313Packages.mlflow</li>
    <li>python313Packages.mlflow.dist</li>
    <li>python313Packages.mmcv</li>
    <li>python313Packages.mmcv.dist</li>
    <li>python313Packages.mmengine</li>
    <li>python313Packages.mmengine.dist</li>
    <li>python313Packages.sagemaker-mlflow</li>
    <li>python313Packages.sagemaker-mlflow.dist</li>
    <li>python313Packages.torchtune</li>
    <li>python313Packages.torchtune.dist</li>
    <li>python314Packages.databricks-sdk</li>
    <li>python314Packages.databricks-sdk.dist</li>
    <li>python314Packages.langchain-azure-dynamic-sessions</li>
    <li>python314Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python314Packages.langchain-classic</li>
    <li>python314Packages.langchain-classic.dist</li>
    <li>python314Packages.langchain-community</li>
    <li>python314Packages.langchain-community.dist</li>
    <li>python314Packages.langchain-deepseek</li>
    <li>python314Packages.langchain-deepseek.dist</li>
    <li>python314Packages.langchain-experimental</li>
    <li>python314Packages.langchain-experimental.dist</li>
    <li>python314Packages.langchain-mongodb</li>
    <li>python314Packages.langchain-mongodb.dist</li>
    <li>python314Packages.langchain-openai</li>
    <li>python314Packages.langchain-openai.dist</li>
    <li>python314Packages.langchain-xai</li>
    <li>python314Packages.langchain-xai.dist</li>
    <li>python314Packages.langgraph-checkpoint-mongodb</li>
    <li>python314Packages.langgraph-checkpoint-mongodb.dist</li>
    <li>python314Packages.langgraph-store-mongodb</li>
    <li>python314Packages.langgraph-store-mongodb.dist</li>
    <li>python314Packages.mlflow</li>
    <li>python314Packages.mlflow.dist</li>
    <li>python314Packages.mmcv</li>
    <li>python314Packages.mmcv.dist</li>
    <li>python314Packages.mmengine</li>
    <li>python314Packages.mmengine.dist</li>
    <li>python314Packages.sagemaker-mlflow</li>
    <li>python314Packages.sagemaker-mlflow.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 96 packages built:</summary>
  <ul>
    <li>langgraph-cli (python313Packages.langgraph-cli)</li>
    <li>langgraph-cli.dist (python313Packages.langgraph-cli.dist)</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.langchain</li>
    <li>python313Packages.langchain-anthropic</li>
    <li>python313Packages.langchain-anthropic.dist</li>
    <li>python313Packages.langchain-aws</li>
    <li>python313Packages.langchain-aws.dist</li>
    <li>python313Packages.langchain-chroma</li>
    <li>python313Packages.langchain-chroma.dist</li>
    <li>python313Packages.langchain-core</li>
    <li>python313Packages.langchain-core.dist</li>
    <li>python313Packages.langchain-fireworks</li>
    <li>python313Packages.langchain-fireworks.dist</li>
    <li>python313Packages.langchain-google-genai</li>
    <li>python313Packages.langchain-google-genai.dist</li>
    <li>python313Packages.langchain-groq</li>
    <li>python313Packages.langchain-groq.dist</li>
    <li>python313Packages.langchain-huggingface</li>
    <li>python313Packages.langchain-huggingface.dist</li>
    <li>python313Packages.langchain-mistralai</li>
    <li>python313Packages.langchain-mistralai.dist</li>
    <li>python313Packages.langchain-ollama</li>
    <li>python313Packages.langchain-ollama.dist</li>
    <li>python313Packages.langchain-perplexity</li>
    <li>python313Packages.langchain-perplexity.dist</li>
    <li>python313Packages.langchain-protocol</li>
    <li>python313Packages.langchain-protocol.dist</li>
    <li>python313Packages.langchain-tests</li>
    <li>python313Packages.langchain-tests.dist</li>
    <li>python313Packages.langchain-text-splitters</li>
    <li>python313Packages.langchain-text-splitters.dist</li>
    <li>python313Packages.langchain.dist</li>
    <li>python313Packages.langgraph</li>
    <li>python313Packages.langgraph-checkpoint</li>
    <li>python313Packages.langgraph-checkpoint-postgres</li>
    <li>python313Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python313Packages.langgraph-checkpoint-sqlite</li>
    <li>python313Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python313Packages.langgraph-checkpoint.dist</li>
    <li>python313Packages.langgraph-prebuilt</li>
    <li>python313Packages.langgraph-prebuilt.dist</li>
    <li>python313Packages.langgraph-runtime-inmem</li>
    <li>python313Packages.langgraph-runtime-inmem.dist</li>
    <li>python313Packages.langgraph.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
    <li>python314Packages.great-tables</li>
    <li>python314Packages.great-tables.dist</li>
    <li>python314Packages.langchain</li>
    <li>python314Packages.langchain-anthropic</li>
    <li>python314Packages.langchain-anthropic.dist</li>
    <li>python314Packages.langchain-aws</li>
    <li>python314Packages.langchain-aws.dist</li>
    <li>python314Packages.langchain-chroma</li>
    <li>python314Packages.langchain-chroma.dist</li>
    <li>python314Packages.langchain-core</li>
    <li>python314Packages.langchain-core.dist</li>
    <li>python314Packages.langchain-fireworks</li>
    <li>python314Packages.langchain-fireworks.dist</li>
    <li>python314Packages.langchain-google-genai</li>
    <li>python314Packages.langchain-google-genai.dist</li>
    <li>python314Packages.langchain-groq</li>
    <li>python314Packages.langchain-groq.dist</li>
    <li>python314Packages.langchain-huggingface</li>
    <li>python314Packages.langchain-huggingface.dist</li>
    <li>python314Packages.langchain-mistralai</li>
    <li>python314Packages.langchain-mistralai.dist</li>
    <li>python314Packages.langchain-ollama</li>
    <li>python314Packages.langchain-ollama.dist</li>
    <li>python314Packages.langchain-perplexity</li>
    <li>python314Packages.langchain-perplexity.dist</li>
    <li>python314Packages.langchain-protocol</li>
    <li>python314Packages.langchain-protocol.dist</li>
    <li>python314Packages.langchain-tests</li>
    <li>python314Packages.langchain-tests.dist</li>
    <li>python314Packages.langchain-text-splitters</li>
    <li>python314Packages.langchain-text-splitters.dist</li>
    <li>python314Packages.langchain.dist</li>
    <li>python314Packages.langgraph</li>
    <li>python314Packages.langgraph-checkpoint</li>
    <li>python314Packages.langgraph-checkpoint-postgres</li>
    <li>python314Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python314Packages.langgraph-checkpoint-sqlite</li>
    <li>python314Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python314Packages.langgraph-checkpoint.dist</li>
    <li>python314Packages.langgraph-cli</li>
    <li>python314Packages.langgraph-cli.dist</li>
    <li>python314Packages.langgraph-prebuilt</li>
    <li>python314Packages.langgraph-prebuilt.dist</li>
    <li>python314Packages.langgraph-runtime-inmem</li>
    <li>python314Packages.langgraph-runtime-inmem.dist</li>
    <li>python314Packages.langgraph.dist</li>
    <li>python314Packages.shiny</li>
    <li>python314Packages.shiny.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>python313Packages.langchain-openai</summary>
<pre>asyncio: mode=Mode.AUTO, debug=False, asyncio_default_fixture_loop_scope=None, asyncio_default_test_loop_scope=function
collecting ... 
collecting 25 items                                                            
collected 380 items / 1 error / 25 deselected / 355 selected                   

==================================== ERRORS ====================================
____ ERROR collecting tests/unit_tests/chat_models/test_responses_stream.py ____
ImportError while importing test module &#x27;/build/source/libs/partners/openai/tests/unit_tests/chat_models/test_responses_stream.py&#x27;.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/0r6k8xa2kgqyp3r4v2w7yrb80ma2iawm-python3-3.13.12/lib/python3.13/importlib/__init__.py:88: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tests/unit_tests/chat_models/test_responses_stream.py:9: in &lt;module&gt;
    from langchain_tests.utils.stream_lifecycle import assert_valid_event_stream
E   ModuleNotFoundError: No module named &#x27;langchain_tests.utils.stream_lifecycle&#x27;
=========================== short test summary info ============================
ERROR tests/unit_tests/chat_models/test_responses_stream.py
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
======================= 25 deselected, 1 error in 1.00s ========================</pre>
</details>
<details>
<summary>python314Packages.langchain-openai</summary>
<pre>____ ERROR collecting tests/unit_tests/chat_models/test_responses_stream.py ____
ImportError while importing test module &#x27;/build/source/libs/partners/openai/tests/unit_tests/chat_models/test_responses_stream.py&#x27;.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/cgk2ykijyaadc1687r0w4psf8pi1fdyc-python3-3.14.3/lib/python3.14/importlib/__init__.py:88: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tests/unit_tests/chat_models/test_responses_stream.py:9: in &lt;module&gt;
    from langchain_tests.utils.stream_lifecycle import assert_valid_event_stream
E   ModuleNotFoundError: No module named &#x27;langchain_tests.utils.stream_lifecycle&#x27;
=============================== warnings summary ===============================
../../../../../nix/store/bfq78kxycndpi0hms86317jjj3y5psbc-python3.14-langchain-core-1.3.2/lib/python3.14/site-packages/langchain_core/_api/deprecation.py:25
  /nix/store/bfq78kxycndpi0hms86317jjj3y5psbc-python3.14-langchain-core-1.3.2/lib/python3.14/site-packages/langchain_core/_api/deprecation.py:25: UserWarning: Core Pydantic V1 functionality isn&#x27;t compatible with Python 3.14 or greater.
    from pydantic.v1.fields import FieldInfo as FieldInfoV1

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
ERROR tests/unit_tests/chat_models/test_responses_stream.py
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
================== 25 deselected, 1 warning, 1 error in 1.07s ==================</pre>
</details>
</details>

--- a/pkgs/development/python-modules/langchain-openai/default.nix
+++ b/pkgs/development/python-modules/langchain-openai/default.nix
@@ -33,14 +33,14 @@
 
 buildPythonPackage (finalAttrs: {
   pname = "langchain-openai";
-  version = "1.1.12";
+  version = "1.2.0";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "langchain-ai";
     repo = "langchain";
     tag = "langchain-openai==${finalAttrs.version}";
-    hash = "sha256-lmEnkwX1tL88xHTrQRe96ZCvU5bM6iwzJ46exI+gfSI=";
+    hash = "sha256-EH4TZpo843m6/7oyFUxt4tkmCiCGD5+GFqM0gfgnWVQ=";
   };
 
   sourceRoot = "${finalAttrs.src.name}/libs/partners/openai";

--- a/pkgs/development/python-modules/langchain-tests/default.nix
+++ b/pkgs/development/python-modules/langchain-tests/default.nix
@@ -28,14 +28,14 @@
 
 buildPythonPackage (finalAttrs: {
   pname = "langchain-tests";
-  version = "1.1.6";
+  version = "1.1.7";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "langchain-ai";
     repo = "langchain";
     tag = "langchain-tests==${finalAttrs.version}";
-    hash = "sha256-W+uJy0t6awNwMpMHEftHV8tDliEhvL/g8V2ZJPWeYk8=";
+    hash = "sha256-zesNbesiLMayvlVrU6NagmE2+o0l6ozwZlOGqvyGpnQ=";
   };
 
   sourceRoot = "${finalAttrs.src.name}/libs/standard-tests";

--- a/pkgs/development/python-modules/langchain-text-splitters/default.nix
+++ b/pkgs/development/python-modules/langchain-text-splitters/default.nix
@@ -21,14 +21,14 @@
 
 buildPythonPackage (finalAttrs: {
   pname = "langchain-text-splitters";
-  version = "1.1.1";
+  version = "1.1.2";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "langchain-ai";
     repo = "langchain";
     tag = "langchain-text-splitters==${finalAttrs.version}";
-    hash = "sha256-I5eMc/E7sCxEQG8+jw3E/M4uB7adKU5IMHRsS6pacsA=";
+    hash = "sha256-AiRl8N2V2UfYLZfqxM8DHZmT76rH19I1gFyOYc/mpYY=";
   };
 
   sourceRoot = "${finalAttrs.src.name}/libs/text-splitters";

